src/Tribe/Editor/Blocks/Tickets.php

```diff
@@ -203,13 +203,20 @@ public function get_tickets( $post_id ) {
 		$all_tickets = Tribe__Tickets__Tickets::get_all_event_tickets( $post_id );
 
 		if ( ! $all_tickets ) {
-			return array();
+			return [];
 		}
 
-		$tickets = array();
+		/** @var Tribe__Tickets__RSVP $rsvp */
+		$rsvp = tribe( 'tickets.rsvp' );
 
+		$tickets = [];
```

Comment:
Should this use `$rsvp->get_tickets( $post_id );` instead now?

Reply:
The description of `\Tribe__Tickets__Editor__Blocks__Tickets::get_tickets()` is:
> Get all tickets for event/post, removing RSVPs

And `\Tribe__Tickets__Editor__Blocks__Rsvp::get_tickets()` is _protected_.

Maybe we should use ORM to get for all active providers other than 'rsvp'???

Reply:
Technically these should all be using the ORM for their specific provider. Can you make that change?